pkg/api/types.go

```diff
@@ -734,6 +734,16 @@ type CustomCloudProfile struct {
 	PortalURL                  string                      `json:"portalURL,omitempty"`
 }
 
+// HasCoreOS returns true if the cluster contains coreos nodes
+func (p *Properties) HasCoreOS() bool {
```

Comment:
Please add this to unit tests

Reply:
let's add a case where it returns true as well

Reply:
whoops -- that's the second case, the message is wrong

Reply:
fixed